### PR TITLE
Add Unit Note & Fee Summary

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#16504

*Description of changes:*
- Add Unit Note view
- Update Fee Summary
- Update breadcrumbs
- Create sub-components for Unit Notes Add and Review

Screenshot 2023-06-08 at 3 47 15 PM

Screenshot 2023-06-08 at 3 47 21 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1377-60gjgt2e.web.app

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Review by orelbn — APPROVED** [review]
Looks great!

File: ppr-ui/src/components/unitNotes/UnitNotesTable.vue
Comment (cameron-eyds) [inline ppr-ui/src/components/unitNotes/UnitNotesTable.vue]: I don't believe you will need this component once you've got my commit 👍

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts
Comment (cameron-eyds) [inline ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts]: We may not need this, the unit note POST is standalone, will not be a part of the registration 👍

File: ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts
Comment (cameron-eyds) [inline ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts]: Same as above.

File: ppr-ui/src/interfaces/mhr-registration-interfaces/MhrUnitNoteIF.ts
Comment (cameron-eyds) [inline ppr-ui/src/interfaces/mhr-registration-interfaces/MhrUnitNoteIF.ts]: The unit notes are not really part of the registration process, stand alone filing per se. 

The latter IF would work for the filing itself though.

**Review by cameron-eyds — COMMENTED** [review]
Looks good although it does go a bit outside the scope of the ticket and will clash a bit with pr: https://github.com/bcgov/ppr/pull/1375.

Was considering you going in first but it might be easier if you take my commit. There is a lot here you probably won't need.

File: ppr-ui/src/store/store.ts
Comment (cameron-eyds) [inline ppr-ui/src/store/store.ts]: Same as above, unit notes are their own filing.
https://amer-api-partner61-demo.apigee.io/docs/mhrproxy/1/routes/mhr/api/v1/notes/%7BmhrNumber%7D/post

**Review by cameron-eyds — APPROVED** [review]
Will approve, your call if you want to merge. 
If you do just keep in mind, there will be content here that is removed as part of the sync up.

File: ppr-ui/src/store/state/state-model.ts
Comment (cameron-eyds) [inline ppr-ui/src/store/state/state-model.ts]: This is good! 
Standalone for sure.

File: ppr-ui/src/store/store.ts
Comment (dimak1) [reply ppr-ui/src/store/store.ts]: This is to get the Notes from the MHR Registrations `GET /mhr/api/v1/registrations/{mhrNumber}`.

Comment (cameron-eyds) [reply ppr-ui/src/store/store.ts]: The state `mhrRegistration` is model for the registration filing itself.

File: ppr-ui/src/components/unitNotes/UnitNotesTable.vue
Comment (dimak1) [reply ppr-ui/src/components/unitNotes/UnitNotesTable.vue]: Yea this would be replaced by your stuff.

File: ppr-ui/src/store/store.ts
Comment (cameron-eyds) [reply ppr-ui/src/store/store.ts]: It could* be stored anywhere for sure and It definitely is a part of the GET due to their coupling to the Registration Record 

Though i feel like the unitNotes(the notes history) should be standalone, in the same manner you did the unitNote (filing state) though due to the fact they are standalone filings and they are mostly just an association/link to an MHR, not part of the original mhrRegistration.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1377-60gjgt2e.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1377-60gjgt2e.web.app

**Review by cameron-eyds — APPROVED** [review]
Nice work! 🎆

**Pull request merged**
